artifacts/config_files/sfall-mods.ini

```diff
@@ -4,8 +4,14 @@
 ;  42 - SHIFT key
 Key=42
 
-; Set to 1 also highlight containers
-Containers=1
+; Set to 1 to also highlight containers
+Containers=0
+
+; Set to 1 to also highlight corpses
+Corpses=0
+
+; Set to 1 to only highlight objects in the player's line-of-sight
+CheckLOS=0
```

Comment:
@Lexx2k , yeah, but this is how it was in previous sfall, so some people might be used to it :)